src/cms/templates/dashboard/region_selection.html

```diff
@@ -0,0 +1,24 @@
+{% extends "authentication/_base.html" %}
+
+{% load i18n %}
+{% load static %}
+
+{% block heading %}
+    {% trans "Select region" %}
+{% endblock %}
+
+{% block content %}
+    <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+        <tbody>
+        {% for region in user_regions %}
```

Comment:
```suggestion
        {% for region in request.user.profile.regions.all %}
```
No need for an extra variable, because the user object is already passed to the template via the [Request Context Processor](https://docs.djangoproject.com/en/3.2/ref/templates/api/#django-template-context-processors-request).